discovery/discovery.yaml

```diff
@@ -234,34 +234,34 @@ components:
         description:
           type: string
           description: Human readable description
-        docsurl:
+        docsUrl:
```

Comment:
Not a big deal, but given a choice I'd vote for "URL" over "Url".

Reply:
Let's discuss in the meeting. I definitely prefer using Pascal case for abbreviations, unless they're only two characters. It makes it clearer when there's a word after it. For example, `docsUrlPage` splits more clearly (IMO) into "docs", "URL", "page" than "docsURLPage" would.

When it comes to two-letter abbreviations (e.g. IO) it gets trickier.

Reply:
you lost me at the word "Pascal" :-)